backend/users/services.py

```diff
@@ -1,6 +1,73 @@
+from typing import OrderedDict
+
+from presentation.models import Presentation, Tag
+from users.models import Profile, User
+
+
 class UserService:
```

Comment:
All these methods don't seem to require self, so you can decorate them with `staticmethod`